src/main/java/org/reaktivity/command/log/internal/LoggableStream.java

```diff
@@ -848,13 +848,13 @@ private void onKafkaMergedDataEx(
         final ArrayFW<KafkaOffsetFW> progress = merged.progress();
 
         out.printf(verboseFormat, index, offset, timestamp,
-                   format("[merged] (%d) %d %s %d %d",
+                   format("[merged] (%d) %d %s %d %d %d",
                            merged.deferred(), merged.timestamp(), asString(key.value()),
-                           partition.partitionId(), partition.partitionOffset()));
+                           partition.partitionId(), partition.partitionOffset(), partition.latestOffset()));
         headers.forEach(h -> out.printf(verboseFormat, index, offset, timestamp,
                                         format("%s: %s", asString(h.name()), asString(h.value()))));
         progress.forEach(p -> out.printf(verboseFormat, index, offset, timestamp,
-                                         format("%d: %d", p.partitionId(), p.partitionOffset())));
+                                         format("%d: %d %d", p.partitionId(), p.partitionOffset(), partition.latestOffset())));
```

Comment:
Should be `p.latestOffset()` to pick up progress latest offset instead of partition latest offset?